### PR TITLE
Initial commit, adding ELK to rancher community catalog

### PR DESCRIPTION
**Pull request by dominikhahn**

### CODE REVIEW

**Comment by cloudnautique** [comment]
Cool, thanks for opening this. 

Since its not necessarily a clear upgrade path from Elasticsearch 1.7.x -> 2.x I'd like to make that clear to users. I'd like to see ES 2 as its own entry. Can you move  the folder from `templates/elasticsearch` to `templates/elasticsearch2`. I'd like to remove the 1.7.x versions of ES, and only include the new 2.2 version.

File: templates/elasticsearch/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: Elasticsearch
```
Comment (cloudnautique) [inline templates/elasticsearch/config.yml]: When moved, this would need to be Elasticsearch 2.x.

File: templates/kibana/0/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+kibana-vip:
```
Comment (cloudnautique) [inline templates/kibana/0/docker-compose.yml]: Lets not include the rancher catalog version in the community catalog as well.

File: templates/logstash/0/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+logstash-indexer-config:
```
Comment (cloudnautique) [inline templates/logstash/0/docker-compose.yml]: Probably don't need the rancher catalog version in the community.

**Comment by roflmao** [comment]
Get errors while trying out the elasticsearch2 templates:

This container keeps dying:
elasticsearch2-rancher_elasticsearch-datanodes_elasticsearch-base-datanode_1

Logs:
```
3/18/2016 2:20:52 PM[2016-03-18 13:20:52,312][WARN ][bootstrap                ] Unable to lock JVM Memory: error=12,reason=Cannot allocate memory
3/18/2016 2:20:52 PM[2016-03-18 13:20:52,313][WARN ][bootstrap                ] This can result in part of the JVM being swapped out.
3/18/2016 2:20:52 PM[2016-03-18 13:20:52,313][WARN ][bootstrap                ] Increase RLIMIT_MEMLOCK, soft limit: 65536, hard limit: 65536
3/18/2016 2:20:52 PM[2016-03-18 13:20:52,313][WARN ][bootstrap                ] These can be adjusted by modifying /etc/security/limits.conf, for example:
3/18/2016 2:20:52 PM	# allow user 'elasticsearch' mlockall
3/18/2016 2:20:52 PM	elasticsearch soft memlock unlimited
3/18/2016 2:20:52 PM	elasticsearch hard memlock unlimited
3/18/2016 2:20:52 PM[2016-03-18 13:20:52,313][WARN ][bootstrap                ] If you are logged in interactively, you will have to re-login for the new limits to take effect.
3/18/2016 2:20:52 PMException in thread "main" java.lang.IllegalStateException: Unable to access 'path.scripts' (/usr/share/elasticsearch/config/scripts)
3/18/2016 2:20:52 PMLikely root cause: java.nio.file.AccessDeniedException: /usr/share/elasticsearch/config/scripts
3/18/2016 2:20:52 PM	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:84)
3/18/2016 2:20:52 PM	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
3/18/2016 2:20:52 PM	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
3/18/2016 2:20:52 PM	at sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:384)
3/18/2016 2:20:52 PM	at java.nio.file.Files.createDirectory(Files.java:674)
3/18/2016 2:20:52 PM	at java.nio.file.Files.createAndCheckIsDirectory(Files.java:781)
3/18/2016 2:20:52 PM	at java.nio.file.Files.createDirectories(Files.java:767)
3/18/2016 2:20:52 PM	at org.elasticsearch.bootstrap.Security.ensureDirectoryExists(Security.java:337)
3/18/2016 2:20:52 PM	at org.elasticsearch.bootstrap.Security.addPath(Security.java:314)
3/18/2016 2:20:52 PM	at org.elasticsearch.bootstrap.Security.addFilePermissions(Security.java:248)
3/18/2016 2:20:52 PM	at org.elasticsearch.bootstrap.Security.createPermissions(Security.java:212)
3/18/2016 2:20:52 PM	at org.elasticsearch.bootstrap.Security.configure(Security.java:118)
3/18/2016 2:20:52 PM	at org.elasticsearch.bootstrap.Bootstrap.setupSecurity(Bootstrap.java:196)
3/18/2016 2:20:52 PM	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:167)
3/18/2016 2:20:52 PM	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:285)
3/18/2016 2:20:52 PM	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:35)
3/18/2016 2:20:52 PMRefer to the log for complete error details.
3/18/2016 2:21:08 PM[2016-03-18 13:21:08,084][WARN ][bootstrap                ] Unable to lock JVM Memory: error=12,reason=Cannot allocate memory
3/18/2016 2:21:08 PM[2016-03-18 13:21:08,085][WARN ][bootstrap                ] This can result in part of the JVM being swapped out.
3/18/2016 2:21:08 PM[2016-03-18 13:21:08,085][WARN ][bootstrap                ] Increase RLIMIT_MEMLOCK, soft limit: 65536, hard limit: 65536
3/18/2016 2:21:08 PM[2016-03-18 13:21:08,085][WARN ][bootstrap                ] These can be adjusted by modifying /etc/security/limits.conf, for example:
3/18/2016 2:21:08 PM	# allow user 'elasticsearch' mlockall
3/18/2016 2:21:08 PM	elasticsearch soft memlock unlimited
3/18/2016 2:21:08 PM	elasticsearch hard memlock unlimited
3/18/2016 2:21:08 PM[2016-03-18 13:21:08,087][WARN ][bootstrap                ] If you are logged in interactively, you will have to re-login for the new limits to take effect.
3/18/2016 2:21:08 PMException in thread "main" java.lang.IllegalStateException: Unable to access 'path.scripts' (/usr/share/elasticsearch/config/scripts)
3/18/2016 2:21:08 PMLikely root cause: java.nio.file.AccessDeniedException: /usr/share/elasticsearch/config/scripts
3/18/2016 2:21:08 PM	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:84)
3/18/2016 2:21:08 PM	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
3/18/2016 2:21:08 PM	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
3/18/2016 2:21:08 PM	at sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:384)
3/18/2016 2:21:08 PM	at java.nio.file.Files.createDirectory(Files.java:674)
3/18/2016 2:21:08 PM	at java.nio.file.Files.createAndCheckIsDirectory(Files.java:781)
3/18/2016 2:21:08 PM	at java.nio.file.Files.createDirectories(Files.java:767)
3/18/2016 2:21:08 PM	at org.elasticsearch.bootstrap.Security.ensureDirectoryExists(Security.java:337)
3/18/2016 2:21:08 PM	at org.elasticsearch.bootstrap.Security.addPath(Security.java:314)
3/18/2016 2:21:08 PM	at org.elasticsearch.bootstrap.Security.addFilePermissions(Security.java:248)
3/18/2016 2:21:08 PM	at org.elasticsearch.bootstrap.Security.createPermissions(Security.java:212)
3/18/2016 2:21:08 PM	at org.elasticsearch.bootstrap.Security.configure(Security.java:118)
3/18/2016 2:21:08 PM	at org.elasticsearch.bootstrap.Bootstrap.setupSecurity(Bootstrap.java:196)
3/18/2016 2:21:08 PM	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:167)
3/18/2016 2:21:08 PM	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:285)
3/18/2016 2:21:08 PM	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:35)
3/18/2016 2:21:08 PMRefer to the log for complete error details.
```

**Pull request closed without merging**